config/hashing.php

```diff
@@ -29,7 +29,7 @@
     */
 
     'bcrypt' => [
-        'rounds' => env('BCRYPT_ROUNDS', 10),
+        'rounds' => env('BCRYPT_ROUNDS', 12),
```

Comment:
Is this a breaking change that might invalidate all existing encryptions? So we don't invalidate the existing logins or similar.

Reply:
Negative. This is password related.

https://github.com/laravel/laravel/pull/6245

> Due to the way hashing works, there are no backwards compatibility issues - older passwords with lower rounds will still be handled properly, and code that automatically rehashes passwords will upgrade them over time. It's also worth pointing out that since rounds are defined in config/hashing.php, existing projects won't automatically get the new rounds cost and thus won't have any performance impacts. The RFC contains hash calculation timings if you'd like more information on the impacts.

I don't think we hash any passwords anywhere seeing as we use OAuth2. If we did, the user controller would still work and would be responsible for checking if the password should be re-hashed, as well as doing that when a weaker hashed password comes along.